### ISSUE TITLE
Need to change buff camp & modify buff terminal

### ISSUE THREAD

**Issue by stacy19201325** [TarkinII/Core3#9]
Need to change the terminal in the buff camp to be a buff terminal instead of blue frog.

Secondly, I'd like to modify the buff terminal to attempt to take from the bank if the player lacks sufficient cash, unless anyone has any objections.

**Comment by stacy19201325** [comment]
Since the buff terminal is done via screenplay, it's not something I can add to a camp.  I CAN make blue frog buffs equal to the "free" buff on the buff terminal, but frog buffs are across-the-board, primary and secondary stats the same.  The buff terminal is not.  Need to discuss this before changes are made.

**Comment by kinshi** [comment]
This needs a design discussion about whether or not we use the blue frog at all. Buff terminal was designed to be a replacement for the frog buffing, and it had been previously discussed that a NPC would be deployed to issue players custom starting gear rather than use the frog for that (better for RP purposes)

The buff terminal buffs like a player would, whereas the frog buffed all stats equally (the blue frog method however is incorrect (not how the live game or player buffing worked, but of course was how it worked on Tarkin

The Blue Frog really should only be about testing, we only used in on 1.0 because it was cheap and easy, however we are better off using more RP-theme ways to do the same things (NPCs)

Action items;

Look at secondary buff values in the buff terminal
Find a replacement for the blue frog in the HTFB

**Comment by Tatwi** [comment]
1. It is possible to add any screenplay function as radial menu option to any existing item, therefore it is possible for the buff terminal to be added to an item placed in a camp.

2. The buff terminal was designed to use whatever value you want for any stat. They are all there in the config table already and are even documented in the table as such.

3. Default SWGEmu does not have a Lua call to take bank credits for the player. It is easy enough to add one (I did for Hondo) and doing so would allow for taking money from both the bank and cash as needed. 

4. For item dispensing, it is nicer to converse with an NPC than to click on an ugly frog model. However, I see no reason to reinvent that wheel if you're just going to keep the frog for skill training. That said, in my original Hondo I added skill training to a different terminal model and I had an NPC merchant system that split those functions apart. The same can be done here if it is desired.
